features/step_definitions/basic_steps.rb

```diff
@@ -10,21 +14,34 @@
   end
 end
 
+When(/^I fill in:$/) do |table|
+  table.hashes.each do |hash|
+      fill_in hash[:element], with: hash[:content]
+  end
+end
+
+When(/^I click the "([^"]*)" button$/) do |button|
+  click_button button
+end
+
+Then(/^I should see "([^"]*)"$/) do |content|
+  expect(page).to have_content content
+end
+
 When(/^I click the link "([^"]*)"$/) do |link|
   click_link(link)
 end
 
-Then(/^I should be on the "([^"]*)" page$/) do |page|
-  expected_page = goto(page)
-  expect(current_path).to eq expected_page
```

Comment:
We had two. We should have removed the other one, actually. I've changed that test to match this one and everything's still running green. We just went through the merge too quickly.